link/uprobe_multi.go

```diff
@@ -0,0 +1,148 @@
+package link
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"unsafe"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/asm"
+	"github.com/cilium/ebpf/internal"
+	"github.com/cilium/ebpf/internal/sys"
+	"github.com/cilium/ebpf/internal/unix"
+)
+
+type UprobeMultiOptions struct {
+	Path          string
+	Offsets       []uint64
```

Comment:
Is this an offset or an address? Judging from `KprobeMultiOptions` I'm guessing `Address`.

Reply:
so both uprobe and uprobe_multi work on top of offsets within the binary file,
perhaps we should rename ex.address to ex.offset

Reply:
uprobe's `Address` was initially called `Offset`, it was renamed for consistency with the Kprobe's API, more context here: https://github.com/cilium/ebpf/commit/5713a12044f2f0f2c4bdf6bd2bd1b9a8cfb77b05.

In the case of `UprobeMulti`, I guess naming it `Offsets` would be a bit confusing (even if from the kernel API perspective is more correct), probably naming it `Addresses` would be better(?)

Reply:
I was wondering why thanks ;-) using opts.Addresses seems better in order to stay consistent with uprobe opts

Reply:
hm, I wonder we could also add opts.Offsets array  with the same functionality as for uprobe's opt.Offset,
at the end the kernel interface is just and offset array